plugins/@grouparoo/mailchimp/src/lib/import/connection.ts

```diff
@@ -10,6 +10,7 @@ const connection: PluginConnection = {
   displayName: "Mailchimp Import Contacts",
   direction: "import",
   description: "Import or update records with data from Mailchimp contacts.",
+  supportIncrementalSchedule: true,
```

Comment:
might be more to do to handle `incremental: false`? unless I missed the changes to `getChangedMembers` 🤔

Reply:
It should be handled! 

https://github.com/grouparoo/grouparoo/blob/08fc49dee423593c56ba44b2f882b05a66081925/plugins/%40grouparoo/mailchimp/src/lib/import/members.ts#L147-L152

The only change would be that if incremental=false, when a run starts, it doesn't get the last run's HWM, so it starts over.  The plugin handles the HWM being null already